scripts/legacy/expand_journal_dataset.py

```
@@ -222,22 +222,22 @@ def create_variation(base_sample: Dict, emotion: str) -> Dict:
+
     # Get templates for this emotion
     templates = emotion_templates.get(emotion, [f"I'm feeling {emotion}."])
-    
+
     # Create variation
     template = random.choice(templates)
-    
+
     # Add some variety to the content
     variations = [
         f"{template} {random.choice(['It\'s been a long day.', 'Things are going well.', 'I need to process this.', 'This is important to me.'])}",
```

Ruff (0.12.2)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)